src/langchain_google_memorystore_redis/vector_store.py

```diff
@@ -408,7 +411,6 @@ def add_texts(
             zip_longest(keys_or_ids, texts, embeddings, metadatas), start=1
         ):
             key, text, embedding, metadata = bundle
-            key = self.key_prefix + key
```

Comment:
why is this line removed?

Reply:
Because prefix is inserted on L401

Reply:
Sorry, read again and found out that if user specify keys_or_ids, the prefix is indeed removed. Added it back in the next commit.